doc/docs/master/enterprise/auth/enable-auth.md

```diff
@@ -1,71 +1,86 @@
-# Configure Access Controls
-
-If access controls are activated, each data repository, or repo,
-in Pachyderm has an Access Control List (ACL) associated with it.
-The ACL includes:
-
-- `READERs` - users who can read the data versioned in the repo.
-- `WRITERs` - users with `READER` access who can also submit
-additions, deletions, or modifications of data into the repo.
-- `OWNERs` - users with READER and WRITER access who can also
-modify the repo's ACL.
-
-Pachyderm defines the following account types:
-
-* **GitHub user** is a user account that is associated with
-a GitHub account and logs in through the GitHub OAuth flow. If you do not
-use any third-party identity provider, you use this option. When a user tries
-to log in with a GitHub account, Pachyderm verifies the identity and
-sends a Pachyderm token for that account.
-* **Robot user** is a user account that logs in with a pach-generated authentication
-token. Typically, you create a user in simplified workflow scenarios, such
-as initial SAML configuration.
-* **Pipeline** is an account that Pachyderm creates for
-data pipelines. Pipelines inherit access control from its creator.
-* **SAML user** is a user account that is associated with a Security Assertion
-Markup Language (SAML) identity provider.
-When a user tries to log in through a SAML ID provider, the system
-confirms the identity, associates
-that identity with a SAML identity provider account, and responds with
-the SAML identity provider token for that user. Pachyderm verifies the token,
-drops it, and creates a new internal token that encapsulates the information
-about the user.
-
-By default, Pachyderm defines one hardcoded group called `admin`.
-Users in the `admin` group can perform any
-action on the cluster including appointing other admins.
-Furthermore, only the cluster admins can manage a repository
-without ACLs.
-
-## Enable Access Control
-
-Before you enable access controls, make sure that
+# Enable Access Controls
+
+Before you can enable access controls, make sure that
 you have activated Pachyderm Enterprise Edition
 as described in [Deploy Enterprise Edition](../deployment.md).
 
+When you initially enable access controls, data access
+is restricted to an *initial admin*, most often yourself.
+This *initial admin* will have `admin` privileges in the
+Pachyderm cluster.
+Then, the `admin` user can configure Pachyderm to work with
+a identity management provider (IdP) of your choice.
+
+If you want to configure a SAML IdP, such as Okta, you need
+to create an initial robot admin account by using the
+`--initial-admin=robot:<user>` flag.
+
+For OIDC users, appointing the initial admin is unnecessary.  
```

Comment:
No, you need to do the same initial admin for OIDC too

Reply:
removed.